test/functional/services/remote/webdriver.ts

```diff
@@ -308,6 +329,11 @@ export async function initWebDriver(
     log.verbose(entry.message);
   });
 
+  // create browser download folder
+  if (!Fs.existsSync(downloadDir)) {
+    Fs.mkdirSync(downloadDir, { recursive: true });
+  }
```

Comment:
Checking for existence isn't necessary, `recursive: true` will only create directories that don't exist.
```suggestion
  Fs.mkdirSync(downloadDir, { recursive: true });
```